Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/sysupdate/ui/OnlineUpdateRegionSelectDialogFragment.java

```diff
@@ -0,0 +1,37 @@
+package org.dolphinemu.dolphinemu.features.sysupdate.ui;
+
+import android.app.Dialog;
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AlertDialog;
+import androidx.fragment.app.DialogFragment;
+import androidx.lifecycle.ViewModelProvider;
+
+import org.dolphinemu.dolphinemu.R;
+
+public class OnlineUpdateRegionSelectDialogFragment extends DialogFragment
+{
+  @NonNull
+  @Override
+  public Dialog onCreateDialog(@Nullable Bundle savedInstanceState)
+  {
+    String[] items = {getString(R.string.current_region), getString(R.string.europe), getString(R.string.japan), getString(R.string.korea), getString(R.string.united_states)};
```

Comment:
DolphinQt doesn't work like this, and it's for a reason. If the user has a system menu install and we let them install a different one, they'll get a messed up system menu with duplicate files. If the user doesn't have a system menu install, letting them select Current Region doesn't make sense.